docs/design/slides/slides-pdf-export.md

```
@@ -0,0 +1,198 @@
+- Animations / transitions in the PDF (export the final rendered
+  state of each slide; ignore `animations`/`transition`).
+- CLI / Node-side export — P0 is browser-only (needs DOM `Image`,
+  `document.fonts`, `OffscreenCanvas`). CLI export is a later phase
+  (would require `node-canvas`).
+- Hyperlinks, bookmarks, PDF metadata beyond title (P1+).
+
+## Proposal Details
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)